INSTALL.md

```diff
@@ -87,6 +88,26 @@ brew update
 brew install homebrew/versions/redis28
 ```
 
+### webdis
+
+Note that this is the only package that assumes that Qiita is already installed (due to library dependencies). Also, that the general suggestion is to have 2 redis servers running, one for webdis/redbiom and the other for Qiita. The default configuration has webdis/redbiom running on the default redis port and Qiita's on 7777.
```

Comment:
Can you add:

"The reason for multiple redis servers is so that the redbiom cache can be flushed without impacting the operation of the qiita server itself."